v3-sql-v4-sql/migrate/helpers/relationHelpers.js

```diff
@@ -158,7 +161,7 @@ async function migrateRelations(tables, relations) {
 
   for (const relation of relations) {
     if (relation.type === 'oneToOne') {
-      await migrateOneToOneRelation(relation);
+      await migrateOneToOneRelation(relation, options);
```

Comment:
If `relation` had an additional type, `modelType` then it might become unnecessary to pass around options.

The model type could possibly be set as part of `processRelation`

https://github.com/strapi/migration-scripts/blob/d660a7ec139d3a85127e0e2ebb44f7046a41c7c8/v3-sql-v4-sql/migrate/helpers/relationHelpers.js#L28

Reply:
Oh nice! I agree - that would be a cleaner solution. Let me give that a try.